packages/desktop-client/src/util/versions.ts

```diff
@@ -15,18 +17,32 @@ function cmpSemanticVersion(
   return x[0] - y[0] || x[1] - y[1] || x[2] - y[2];
 }
 
-export async function getLatestVersion(): Promise<string> {
-  let response = await fetch(
-    'https://api.github.com/repos/actualbudget/actual/tags',
-  );
-  let json = await response.json();
-  let tags = json.map(t => t.name).concat([`v${window.Actual.ACTUAL_VERSION}`]);
-  tags.sort(cmpSemanticVersion);
+export async function getLatestVersion(): Promise<string | 'unknown'> {
```

Comment:
Generally two changes here:
1. if using playwright tests - skip this request and just resolved to a fake version
2. if github api fails - gracefully fall back; currently the app just blows up which is not great..